src/config.rs

```diff
@@ -392,6 +398,12 @@ impl Config {
             .help("Keep RRDP responses in the given directory")
             .takes_value(true)
         )
+        .arg(Arg::with_name("max-object-size")
+            .long("max-object-size")
+            .value_name("BYTES")
+            .help("Maximum size of downloaded objects (0 .. no limit)")
```

Comment:
This help message seems strange, like it allows to pass anything between 0 and unlimited, while 0 actually should be used for unlimited. Something like `(0 means no limit)` perhaps?